### PR TITLE
Update changelog to 3.5.3

### PR DESCRIPTION
**Pull request by jeanbisutti**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # CHANGELOG
 
+## Version 3.5.3 GA (05/21/2024)
+
+### Enhancements:
+* Display a message about sampling at start-up
+  ([#3676](https://github.com/microsoft/ApplicationInsights-Java/pull/3676))
```
Comment (trask) [inline CHANGELOG.md:7]:
```suggestion
* Emit stable OpenTelemetry HTTP metrics
  ([#3676](https://github.com/microsoft/ApplicationInsights-Java/pull/3676))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+* Support clientsecret via env var
+  ([#3692](https://github.com/microsoft/ApplicationInsights-Java/pull/3692))
+* Display a message about sampling at start-up
+  ([#3700](https://github.com/microsoft/ApplicationInsights-Java/pull/3700))
+* Fail if HTTP status code used in sampling override
+  ([#3708](https://github.com/microsoft/ApplicationInsights-Java/pull/3708))
+* Fix misleading connection string message
+  ([#3709](https://github.com/microsoft/ApplicationInsights-Java/pull/3709))
+* Update to OpenTelemetry Java Instrumentation 2.4.0
```
Comment (heyams) [inline CHANGELOG.md:16]:
```suggestion
* Update to OpenTelemetry Java 2.4.0
```

Comment (jeanbisutti) [reply CHANGELOG.md:16]: As discussed, 2.4.0 is the instrumentation version (not the OTel Java SDK version).

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
 # CHANGELOG
 
+## Version 3.5.3 GA (05/24/2024)
+
+### Enhancements:
+* Emit stable OpenTelemetry HTTP metrics
+  ([#3676](https://github.com/microsoft/ApplicationInsights-Java/pull/3676))
+* Support clientsecret via env var
```
Comment (trask) [inline CHANGELOG.md:8]:
```suggestion
* Support AAD clientsecret via env var
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+  ([#3700](https://github.com/microsoft/ApplicationInsights-Java/pull/3700))
+* Fail if HTTP status code used in sampling override
+  ([#3708](https://github.com/microsoft/ApplicationInsights-Java/pull/3708))
+* Fix misleading connection string message
+  ([#3709](https://github.com/microsoft/ApplicationInsights-Java/pull/3709))
+* Update to OpenTelemetry Java Instrumentation 2.4.0
+  ([#3711](https://github.com/microsoft/ApplicationInsights-Java/pull/3711))
+* Introduce explicit option to disable ingestion sampling
+  ([#3713](https://github.com/microsoft/ApplicationInsights-Java/pull/3713))
+* Emit `_APPRESOURCEPREVIEW_` custom metrics only in Application Insights private preview integration for AKS
```
Comment (trask) [inline CHANGELOG.md:20]:
```suggestion
* Remove `_APPRESOURCEPREVIEW_` custom metric as it is still in preview
```

**Review by trask — APPROVED** [review]

**Pull request merged**